__tests__/app.test.js

```diff
@@ -106,18 +100,51 @@ describe("GET/api/articles/:article_id", () => {
       .get("/api/articles/4")
       .then((response) => {
         const article = response.body.article;
-        expect(response.status).toBe(200);
-        expect(article).toEqual({ article:
-          { article_id: 4,
+        expect(200);
+        expect(article).toEqual({
+          article: {
+            article_id: 4,
             title: "Student SUES Mitch!",
             topic: "mitch",
             author: "rogersop",
             body: "We all love Mitch and his wonderful, unique typing style. However, the volume of his typing has ALLEGEDLY burst another students eardrums, and they are now suing for damages",
-            "created_at": "2020-05-06T01:14:00.000Z",
+            created_at: "2020-05-06T01:14:00.000Z",
             article_img_url:
               "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
-              votes: 0
-          }})
+            votes: 0,
+          },
+        });
+      });
+  });
+});
+describe("GET/api", () => {
+  test("returns status 200 and an object", () => {
+    return request(app)
+      .get("/api")
+      .then((response) => {
+        const endPoints = response.body.endPoints;
+        expect(200);
+        expect(typeof endPoints).toEqual("object");
       });
-  })
+  });
+  test("returns 200 and an array of objects with the objects containing correct keys", () => {
+    return request(app)
+      .get("/api/articles")
+      .then((response) => {
+        const articles = response.body.articles;
+        expect(200);
+        expect(Array.isArray(articles)).toEqual(true);
+        expect(articles[0]).toEqual({
+          author: "icellusedkars",
+          title: "Eight pug gifs that remind me of mitch",
+          article_id: 3,
+          topic: "mitch",
+          created_at: "2020-11-03T09:12:00.000Z",
+          votes: 0,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: "2",
+        });
+      });
+  });
```

Comment:
So for this test, you're only checking that the first article is the exact one.

When doing an api call where you're going to get many results in the array, it's better to test the 'shape' of the data.

For example here: check that the `articles` on the response body is an array, check it's length, and then use something like a .forEach to check each key has a particular data type, i.e. every article_id should be a number, every title should be a string and so on.